test/fixtures/worker-runner/index.js

```diff
@@ -0,0 +1,16 @@
+const pipe = Pear.worker.pipe()
+
+pipe.once('data', (workerPath) => {
+  const workerPipe = Pear.worker.run(workerPath)
+
+  const messages = []
```

Comment:
in the new convention, worker should be very simple and specific to the test (we can create as many workers as we want)

simple here means: worker sends a specific message (no poor man rpc, no message string check)

e.g. these workers send one message only
- https://github.com/holepunchto/pear/blob/main/test/fixtures/dht-bootstrap/index.js
- https://github.com/holepunchto/pear/blob/main/test/fixtures/require-assets/index.js
- https://github.com/holepunchto/pear/blob/teardown-3/test/fixtures/teardown/index.js

Reply:
Ah, this was modeled around the old worker test. Will create a separate one that's simpler.